src/drives/manual_request.rs

```diff
@@ -34,3 +34,30 @@ impl DrivesItemsIdApiClient {
         params: file_name
     );
 }
+
+impl WorksheetsIdApiClient {
+    patch!(
+        doc: "Invoke function range",
+        name: update_range_object_by_address,
+        path: "/worksheets/{{RID}}/range(address='{{id}}')",
+        body: true,
+        params: address
+    );
+}
```

Comment:
I wasn't gonna say anything cause its not that big of a deal, but since there are changes needed could you go ahead and add an empty space between the two impl blocks here. Just to keep everything nicely formatted.
```suggestion
}

```

Reply:
All requested changes have been made.

Thanks for taking the time to provide feedback.